flags.go

```diff
@@ -50,10 +50,15 @@ var (
 		Name:  "tlsverify",
 		Usage: "Use TLS and verify the remote",
 	}
+	flOverCommit = cli.IntFlag{
+		Name:  "overcommit, oc",
+		Usage: "OverCommit to apply on resources",
+		Value: 105,
```

Comment:
Wouldn't this be better if we prevent under committing? Overcommit actually takes the percentage of resources to overcommit, and in the code we simply overcommit + 100?